Sources/Public/Controls/AnimatedControl.swift

```diff
@@ -100,6 +109,49 @@ open class AnimatedControl: UIControl {
     super.cancelTracking(with: event)
   }
 
+  #elseif canImport(AppKit)
+  open override func mouseDown(with _: NSEvent) {
+    guard let window = window else { return }
+
+    currentState = .highlighted
+    updateForState()
+
+    // AppKit mouse-tracking loop from:
+    // https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/EventOverview/HandlingMouseEvents/HandlingMouseEvents.html#//apple_ref/doc/uid/10000060i-CH6-SW4
```

Comment:
this is really how `NSControl` works????? so weird man

Reply:
Weird API :/